app/forms/claim/mentor_form.rb

```diff
@@ -0,0 +1,41 @@
+class Claim::MentorForm < ApplicationForm
```

Comment:
Oh, I forgot to call out this one. Can we rename this to `Claim::MentorsForm` please to show that this form works with multiple mentors, not just a singular mentor.

Reply:
Yes, I forgot this as well 👍🏻